config/default.yml

```diff
@@ -1757,16 +1757,13 @@ Lint/RedundantSafeNavigation:
   Description: 'Checks for redundant safe navigation calls.'
   Enabled: pending
   VersionAdded: '0.93'
-  Safe: false
```

Comment:
I would keep it as `Safe: false` given that there are some cases can't handle perfectly.